mantle/platform/qemu.go

```diff
@@ -306,6 +306,41 @@ func (inst *QemuInstance) SwitchBootOrder() (err error) {
 	return nil
 }
 
+// RemoveBlockDevice deletes the primary device from a qemu instance
+// and sets the seconday device as primary.
+func (inst *QemuInstance) RemoveBlockDevice() error {
+	var primaryDevice string
+	var secondaryDevicePath []string
```

Comment:
Any reason this couldn't just be a string given we'd only ever use a single entry from the list?

Reply:
Since I was using a loop, so thought of keeping all entries. but yeah, makes sense to me.